iphone/Classes/WebFont.m

```diff
@@ -151,9 +151,9 @@ -(UIFont *) font
                 }
             }
             if (font == nil) {
-                //NO valid family specified. Just check for characteristics. Semi bold is ignored here.
+                //NO valid family specified. Just check for characteristics.
                 if (self.isBoldWeight) {
-                    UIFont* theFont = [UIFont boldSystemFontOfSize:self.size];
+                    UIFont* theFont = [UIFont systemFontOfSize:self.size weight:UIFontWeightBold];
```

Comment:
This API is available in iOS 8.2 and above only. Need to put check for same.
+ (UIFont *)systemFontOfSize:(CGFloat)fontSize weight:(CGFloat)weight NS_AVAILABLE_IOS(8_2);